quick_start/more.rst

```diff
@@ -0,0 +1,278 @@
+
+Now we can write our ``predict`` method. The second argument, ``inputArr``, is the input -- in this case, our four ``Double``\s. To return text, we simply return from the method in Python.
+
+.. code :: python
+
+    def predict(self, inputArr):
+        points = [inputArr]
+        df = pd.DataFrame(points, columns=self.cols)
+
+        results = self.rf.predict(df)
+        return results.item()
+
+Configuration
+*************
+
+When your module is started, it is run in a Linux container on the NStack server. Because our module uses libraries like ``pandas`` and ``sklearn``, we have to tell NStack to install some extra operating system libraries inside your module's container. NStack lets us specify these in our ``nstack.yaml``` configuration file in the ``packages`` section. Let's add the following packages:
+
+.. code :: yaml
+
+    packages: ['numpy', 'python3-scikit-learn.x86_64', 'scipy', 'python3-scikit-image.x86_64', 'python3-pandas.x86_64']
+
+Additionally, we want to tell NStack to copy our ``train.csv`` file into our module, so we can use it to train our data. ``nstack.yaml`` also has a section for specifying files you'd like to include:
+
+.. code :: yaml
+
+    files: ['train.csv']
+
+
+Publishing and Starting
+***********************
+
+Now we're ready to build and publish our classifier. Remember, even though we run this command locally, our module gets built and published to your NStack Server.
+
+.. code :: bash
+
+    ~/irisclassify/ $ nstack build
+    Building NStack Container module irisclassify. Please wait. This may take some time.
+    Module irisclassify built successfully. Use `nstack list methods` to see all available methods.
+
+We can see our method, ``irisclassify.predict``. Including our ``demo.numChars`` method from the previous tutorial, we should now have two:
+
+.. code :: bash
+ 
+   ~/irisclassify/ $ nstack list methods
+    irisclassify.predict : (Double, Double, Double, Double) -> Text
+    demo.numChars : Text -> Integer
+
+Our classifier is now published, but to use it we need to connect it to an event-source and sink. In the previous tutorial, we used HTTP as a source, and the NStack log as a sink. We can do the same here by starting the following workflow.
+
+.. code :: bash
+   
+    ~/irisclassify/ $ nstack start "source(http:///irisendpoint : (Double, Double, Double, Double)) | irisclassify.predict | sink(log:// : Text)"
+
+This creates an HTTP endpoint on ``http://localhost:8080/irisendpoint`` which can receive four ``Double``\s, and writes the results to the log as ``Text``. We can test our classifier by sending it some of the sample data from ``train.csv``:
+
+.. code :: bash
+
+   ~/irisclassify/ $ curl -X PUT -d '{ "params" : [4.7, 1.4, 6.1, 2.9] }' localhost:8080/irisendpoint 
+   Success
+   ~/irisclassify/ $ nstack log 2  
+   Feb 17 10:32:30 nostromo nstack-server[8925]: OUTPUT: "Iris-versicolor"
+
+Great! Our classifiier is now productionised.
+
+Other Sources and Sinks
```

Comment:
I wonder if a better header here would be `Connecting to a Database` - it's more user-story focused, perhaps more compelling?